overlord/snapstate/snapstate_remove_test.go

```diff
@@ -1758,6 +1758,14 @@ func (s *validationSetsSuite) SetUpTest(c *C) {
 		snapstate.AutomaticSnapshot = oldAutomaticSnapshot
 	})
```

Comment:
I missed where this was located, I understand that we started with remove, but maybe this suite if we want to share only  one, should be defined in snapstate_install_test.go ?